package/gluon-respondd/src/respondd.c

```diff
@@ -211,11 +212,21 @@ static struct json_object * get_rootfs_usage(void) {
 	return jso;
 }
 
+static struct json_object * get_time(void) {
+	struct timespec now;
+	clock_gettime(CLOCK_REALTIME, &now);
```

Comment:
Maybe EINVAL could happen here. I think EFAULT and EPERM are definitely no problem.

Reply:
yes, EINVAL could happen, but in that case nothing really breaks. The time will be invalid of course but that is fine, I guess....

Reply:
actually... when EINVAL happens, the field should not exist in the json. This I could do. 

Is that what you meant, @lemoer?

Reply:
We should just not add a time when *clock_gettime* fails (adding NULL would be fine, it shouldn't actually happen anyways).

Reply:
I think this is better. Another way would be to initialize, so it is not leaking any random stuff.